week-1/Homework/extra/2-piping.js

```diff
@@ -1,58 +1,58 @@
-/*
-  PIPING FUNCTIONS
-  ================
-  1. Write 3 functions:
-  - one that adds 2 numbers together
-  - one that multiplies 2 numbers together
-  - one that formats a number so it's returned as a string with a £ sign before it (e.g. 20 -> £20)
-
-  2. Using the variable startingValue as input, perform the following operations using your functions all
-  on one line (assign the result to the variable badCode):
-  - add 10 to startingValue
-  - multiply the result by 2
-  - format it
-  
-  3. Write a more readable version of what you wrote in step 2 under the BETTER PRACTICE comment. Assign
-  the final result to the variable goodCode
-*/
-
-function add() {
-
+// PIPING FUNCTIONS
+//   ===
+//   === === === === =
+//   1. Write 3 functions:
+//   -one that adds 2 numbers together -
+//   one that multiplies 2 numbers together -
+//   one that formats a number so it’ s returned as a string with a£ sign before it(e.g.20 - > £20)
+// 2. Using the variable startingValue as input, perform the following operations using your functions all
+// on one line(assign the result to the variable badCode):
+//   -add 10 to startingValue -
+//   multiply the result by 2 -
+//   format it
+// 3. Write a more readable version of what you wrote in step 2 under the BETTER PRACTICE comment.Assign
+// the final result to the variable goodCode
+
+
+function add(num1, num2) {
+  const total = num1 + num2
+  return Number(total.toFixed(2))
 }
 
-function multiply() {
-
+function multiply(num1, num2) {
+  const total = num1 * num2
+  return Number(total.toFixed(2))
 }
 
-function format() {
-
+function format(num) {
+  return "£" + num
 }
 
-const startingValue = 2
+// const startingValue = 2
 
-// Why can this code be seen as bad practice? Comment your answer.
-let badCode = 
+// // Why can this code be seen as bad practice? Comment your answer.
+// let badCode =
```

Comment:
You've got the functions right, and they work - can you add in some code here which calls the functions, to prove that they work and make the tests pass? :) As a reminder of how functions work, you can write:
`format(multiply(2, 5))` to call multiply, and then format the return value of the function.